EIPS/eip-4824.md

```diff
@@ -0,0 +1,220 @@
+---
+eip: 4824
+title: Decentralized Autonomous Organizations
```

Comment:
Suggest updating the name to `Metadata Standard of Decentralized Autonomous Organizations` to reflect the general scope of this proposal

Reply:
See response below on your other name change suggestion!

Reply:
@thelastjosh this title will likely cause confusion among many people. I would like to request that you and your coauthors come up with a more specific title regarding the proposal.

Reply:
@lightclient Changed.